crates/holochain/src/core/workflow/sys_validation_workflow/validation_query.rs

```diff
@@ -110,9 +110,7 @@ async fn get_ops_to_validate(
     db.read_async(move |txn| {
         let mut stmt = txn.prepare(&sql)?;
         let r = stmt.query_and_then([], |row| {
-            let op = WorkflowResult::Ok(holochain_state::query::map_sql_dht_op(
-                true, "dht_type", row,
-            )?)?;
+            let op = holochain_state::query::map_sql_dht_op(true, "dht_type", row)?;
```

Comment:
The extra `Ok` and `?` weren't doing anything useful. The error can be converted directly.